doc/can-query-logic.md

```diff
@@ -14,139 +14,141 @@ data caching and real-time behavior.
 
 @signature `new QueryLogic( [schemaOrType] [,options] )`
 
-The `can-query-logic` package exports a constructor function that builds _query logic_
-from:
+  The `can-query-logic` package exports a constructor function that builds _query logic_
+  from:
 
-- an optional schema or type argument, and
-- an optional `options` argument used to convert alternate parameters to
+  - an optional schema or type argument, and
+  - an optional `options` argument used to convert alternate parameters to
   the expected [can-query-logic/query] format.
 
 
-For example, the following builds _query logic_ from a [can-define/map/map]:
+  For example, the following builds _query logic_ from a [can-define/map/map]:
 
-```js
-import {DefineMap, QueryLogic} from "can";
+  @sourceref ./can-query-logic-models/todo-example.js
```

Comment:
it's odds that `const todoQueryLogic = new QueryLogic(Todo);` isn't highlighted as that's the most important part.